auditwheel/wheel_abi.py

```diff
@@ -112,7 +113,38 @@ def analyze_wheel_abi(wheel_fn: str):
     log.debug('external reference info')
     log.debug(json.dumps(external_refs, indent=4))
 
+    # get versioned symbols from external_refs
+    # since linux tag (priority 0) has no white-list, do not analyze it
+    external_ref_no_linux = dict(set([(v, k) for e in external_refs.values() if e['priority'] != 0 for k, v in e['libs'].items() if v is not None]))
+    external_refs_symbol_policy = {}
+    log.debug(external_ref_no_linux)
+    for path, elf in elf_file_filter([path for path in iter(external_ref_no_linux)]):
```

Comment:
`[path for path in iter(...)]` can be spelled `list(...)`. Or `.keys()` would work, I think.